src/declaration.h

```diff
@@ -580,10 +580,10 @@ class FuncDeclaration : public Declaration
                                         // scopes from having the same name
     VarDeclaration *vthis;              // 'this' parameter (member and nested)
     VarDeclaration *v_arguments;        // '_arguments' parameter
-#ifdef IN_GCC
+#ifdef IN_GCC // %%
```

Comment:
Oh right, I misread this block as a `#ifndef IN_GCC` block.  I'm considering doing the same in dmd.